src/lib/utils/route-data.ts

```diff
@@ -1,14 +1,5 @@
-import { L10nTranslationService } from 'angular-l10n';
-import { Data, Params, Route, Routes } from '@angular/router';
-
-/** Type of method that returns a label based on the activated route's params, data and queryParams.
- * angular-l10n's L10nTranslationService can be used to provide multi-lingual labels */
-export type LabelFunction = (
-    translation: L10nTranslationService,
-    params: Params,
-    routeData: Data,
-    queryParams: Params
-) => string;
+import { LabelFunction } from '../route-data/route-data-function-types';
+import { Route, Routes } from '@angular/router';
 
 export interface RouteDataInterface {
```

Comment:
das interface hast du bereits in der anderen Datei definiert. Hier muss es raus.

Reply:
Der type `RouteData` könnte ggf. auch in die Datei.